components/story-inquiry-page/StoryInquiryPage.styles.js

```diff
@@ -0,0 +1,164 @@
+import styled from 'styled-components';
+import { min } from '../../lib/responsive';
+
+const LongLinesDesktop = styled.img`
+  display: none;
+  @media ${min.desktop} {
+    display: block;
+    position: relative;
+    width: 11.2em;
+    margin: 0 0 0 5.8em;
+  }
+`;
+
+const LongLinesFlippedDesktop = styled.img`
+display: none;
+@media ${min.desktop} {
+  display: block;
+  position: relative;
+  width: 11.2em;
+  margin: -32em 0 0 4.8em;
+}
+}
+`;
+
+const LongLinesTablet = styled.img`
+  @media ${min.tablet} {
+    display: block;
+    position: relative;
+    width: 7em;
+    margin: 0 0 0 0.9em;
+  }
+  @media ${min.desktop} {
+    display: none;
+  }
+`;
+
+const LongLinesFlippedTablet = styled.img`
+  @media ${min.tablet} {
+    display: block;
+    position: relative;
+    width: 7em;
```

Comment:
nit & optional: everything in this component is the same as **LongLinesTablet** except for margin, maybe you could do:

```
const LongLinesFlippedTablet = styled(LongLinesTablet)`
    @media ${min.tablet} {
        margin: 0 0 0 0.9em;
   }
`;
```